src/clients/base-evernode-client.js

```diff
@@ -774,6 +774,25 @@ class BaseEvernodeClient {
         return null;
     }
 
+    /**
+     * 
+     * @param {string} ownerAddress | Address of the owner
+     * @returns An array of candidate information. Returns empty array if no candidates;
+     */
+    async getDudHostCandidatesByOwner(ownerAddress = this.xrplAcc.address) {
+        console.log(ownerAddress)
+        try {
+            let candidates = await this.#firestoreHandler.getCandidates({ownerAddress: ownerAddress});
+            if(candidates && candidates.length > 0) {
+                candidates = candidates.filter(c => c.uniqueId[9] == '3');
```

Comment:
What if we use this comparison `StateHelpers.getCandidateType(c.uniqueId) === EvernodeConstants.CandidateTypes.DudHost` ?